### PR TITLE
Feature react component - Calculator structure

### PR DESCRIPTION
**Pull request by vikipretium**
## Requirements 

- Delete all the boilerplate from CRA (text, images, styles). ✔️ 
- Create a directory called components. ✔️ 
- Inside components, create a new Calculator.js file. ✔️ 
- In Calculator.js, create a React component that matches this design: ✔️ 
- Use class-based components. ✔️
-  Once the Calculator is ready, import it and use it in your main component (App.js). ✔️
- [](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#need-a-big-picture)Test it in the browser; it should render without problems. ✔️ 

image

Thanks for reviewing my pull request. 🙏🏼 
I apologise for the delay 🙏🏼

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+// import './Calculator.css';
```
Comment (mikethreels) [inline src/components/Calculator.js]: - [x] please remove any commented out code this will make your project look unprofessional and harder to read.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi V, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._
- [x] Please delete All the boilerplate that got created by the CRA (texts, images, and styles) this includes for example "App.css", "App.test.js", "index.css", "logo.csv", "reportWebVitals.js", "screenshot.png", "setupTests.js" and "test.gif"
- [x] please make sure to use[ the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse, including the links (for example to the MIT license). keep in mind that all sections of the template are mandatory except for the "Live Demo", "Run Tests" and "Deployment" sections, these are only mandatory if the specific project you're working on requires it.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (vikipretium) [reply src/components/Calculator.js]: I am so sorry @mikethreels. I didn't noticed it. I'll fix it 👍🏼 
Thank you 👍🏼

**Review by elyor-doniyorov — CHANGES_REQUESTED** [review]
Hi @vikipretium, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
- [x] You still have boilerplates created by CRA (texts, images, and styles) in your `public `and `src `folders. Kindly remove them because they make your codebase unclean.
- [x] Also update your Readme file as requested by the previous reviewer. Kindly take a look at the below screenshot of the previous review comments 👇 
Screenshot (240)



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by elyor-doniyorov — APPROVED** [review]
Hi @vikipretium, 

In my opinion, your project milestone is complete!  There is nothing else to say other than... well done :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by vikipretium** [comment]
Thank you @elyor-doniyorov.
Please Let me know If I need to add anything else, in case I missed it to add. 
Today I didn't checked the review properly. 
If i have seen about the readme and boilerplate removal changes. I would have fixed in my second review request itself.
Thanks for approving my milestone project @elyor-doniyorov 👍🏼

**Pull request merged**